context/app/static/js/pages/entitySearch/utils.js

```diff
@@ -0,0 +1,42 @@
+import { FILTER_TYPES } from 'js/components/entitySearch/filters/Filter/enums';
+
+function getFilterProps({ field, ...rest }) {
+  return {
+    componentId: `${field}-filter`,
+    dataField: field,
+    ...rest,
+  };
+}
+
+function getFieldProps({ field, title }) {
+  return {
+    field,
+    title,
+  };
+}
+
+function getDonorMetadataFilters(isDonor) {
+  const bmiField = 'body_mass_index_value';
+  const ageField = 'age_value';
+  const pathPrefix = isDonor ? '' : 'donor.';
+  const labelPrefix = isDonor ? '' : 'Donor ';
+
+  return [
+    { field: `${pathPrefix}mapped_metadata.sex`, title: `${labelPrefix}Sex`, type: FILTER_TYPES.multiList },
```

Comment:
This may be a stop-gap: If so, never mind, but I'd like to distinguish configurations from the rest of the code, if possible.